mscore/loginmanager.cpp

```diff
@@ -418,6 +418,8 @@ void LoginManager::onGetMediaUrlRequestReady(QByteArray ba)
                         }
                   }
             preferences.exportMp3BitRate = br;
+            // delete the temp file created above
+            QFile::remove(path);
```

Comment:
That will not work. It needs to be deleted in mediaUploadFinished or mediaUploadError or just let the OS delete it. That's why it's in  temp directory